### DISCUSSION TITLE
Netbean failed to open project folder

### DISCUSSION THREAD

**Discussion by JinWoo-dev** [apache/netbeans#6050] [Q&A] [answered]
Hi guys,

Need help on this error. My netbean failed to open project folder & it's searching for non-exist shortcut...
This is the error:
![image](https://github.com/apache/netbeans/assets/99469095/fec12dc4-5d35-48dd-a535-dbd23a9029f5)

Believe it or not, my netbean is newly installed & doesn't has any project on it.
![image](https://github.com/apache/netbeans/assets/99469095/db4bd140-3c4a-4703-be3f-440f93d2239a)

Need help...

**Comment by asbachb** [comment]
Have you a file with that name mentioned in that message in your project folder?

**Comment by JinWoo-dev** [comment]
I don't have that file. That file is not even in my project.

**Reply by asbachb** [reply]
For me it's not 100% clear what happens.

You open up NetBeans. The message is shown immediately or you try to open an existing project? Where is this project coming from?

Maybe you could write stept by step what you're doing until you get to that message.

**Comment by JinWoo-dev** [comment]
Last time, I install netbean in D: drive. Recently, my D: drive got
corrupted and all my projects are gone. When I install a new netbean, I
want to open an existing project and it's showing the error. This error
also occur when I'm clicking the browse button in netbean.

**Comment by emaphis** [comment, marked as answer]
When you reinstall NetBeans don't have it copy over old settings.  It may be something is corrupt.

**Comment by JinWoo-dev** [comment]
Ok, I'll check it out.
